lib/lndclient/LndClient.ts

```diff
@@ -34,7 +40,7 @@ class LndClient extends BaseClient{
       const lndCert = fs.readFileSync(`${datadir}tls.cert`);
       const credentials = grpc.credentials.createSsl(lndCert);
       const lnrpcDescriptor: any = grpc.load(rpcprotopath);
-      this.lightning = new lnrpcDescriptor.lnrpc.Lightning('127.0.0.1:10009', credentials);
+      this.lightning = new lnrpcDescriptor.lnrpc.Lightning(this.host + ':' + this.port, credentials);
```

Comment:
Can just use `host` and `port` here without `this`.

Also, although tslint doesn't enforce the [prefer-template](https://eslint.org/docs/rules/prefer-template) rule, I think we might as well stick to using it since that's how it is everywhere else in the code. So use `${host}:${port}` surrounded by backticks here instead of the string concatenation with the `+` operator.